### PR TITLE
List customer without status locked

### PR DESCRIPTION
**Pull request by pthegner**
# Description

This PR filter customer on the locked status

## Issue

Issue link: BOT-1179, BOT-1151

## How to test

Here are the following steps to test this pull request:

- change a customer settings on cus_locked field to True (database)
- connect into admin nmm on the customer list
- you should not see the customer that you have changed

### CODE REVIEW

**Comment by Fahmus** [comment]
Why you don't use:
`->findBy(array('locked'=> false));`
In CustomerController::listAction
?

**Comment by pthegner** [comment]
retest this please

File: Entity/CustomerRepository.php [hunk 1]
```diff
@@ -12,6 +12,12 @@
  */
 class CustomerRepository extends EntityRepository
 {
+    private $filter = array('locked' => false);
+
+    public function findAll() {
+        return $this->findBy($this->filter);
```
Comment (Netmisa) [inline Entity/CustomerRepository.php]: Why you doesn't use findBy directly ?
`findAll` != `findBy($this->filter)`

File: Entity/CustomerRepository.php [hunk 2]
```diff
@@ -12,6 +12,22 @@
  */
 class CustomerRepository extends EntityRepository
 {
+    private $filter = array('locked' => false);
+
+    public function findAll() {
+        return parent::findBy($this->filter);
```
Comment (Netmisa) [inline Entity/CustomerRepository.php]: Why you doesn't use findBy directly ?
`findAll` != `findBy($this->filter)`

**Comment by Fahmus** [comment]
jenkins, retest this please

**Pull request merged**